site/content/docs/5.3/components/orange-navbar.md

```diff
@@ -356,8 +356,9 @@ You can add a search input into your Global header.
       ...
 
       <!-- Navbar with action icons -->
-      <form class="d-flex d-none d-lg-flex col-3 navbar-item" role="search">
+      <form class="d-none d-lg-flex col-3 navbar-item" role="search">
```

Comment:
Not sure about this one because `"search"` is already an implicit role for `form` element.
I would recommend to put the form inside the div with a `role="search"` attribute (see for instance https://adrianroselli.com/2015/08/where-to-put-your-search-role.html).

I am just wondering if this sort of 'a11y expert trick' is too much for our doc or not?  🤔

Reply:
I think it's fine and better to do it this way. We may need to bring it back in Bs.